libioc/Config/Jail/Defaults.py

```diff
@@ -85,22 +103,38 @@
     "exec_prestop": None,
     "exec_stop": "/bin/sh /etc/rc.shutdown",
     "exec_poststop": None,
+    "exec_system_user": False,
     "exec_jail_user": "root",
+    "exec_system_jail_user": False,
     "exec_timeout": "600",
     "stop_timeout": "30",
     "mount_procfs": "0",
     "mount_devfs": "1",
     "mount_fdescfs": "0",
+    "mount_linprocfs": "0",
+    "mountpoint": "0",
+    "notes": None,
```

Comment:
This appears to be yet another way to leave a `comment`. Both should go into the `user.*` namespace.